cls/SourceControl/Git/Change.cls

```diff
@@ -131,8 +131,14 @@ ClassMethod RefreshUncommitted(Display = 0, IncludeRevert = 0, Output gitFiles,
         set examine=$select(action="add":1,action="edit":1,action="delete":1, IncludeRevert&&(action="revert"):1,1:0)
         if 'examine set filename=$order(tFileList(filename),1,action) continue
 
+        set context = ##class(SourceControl.Git.PackageManagerContext).%Get()
+        set packageRoot = context.Package.Root
         set InternalName = ##class(SourceControl.Git.Utils).NameToInternalName(filename,0,0)
 
+        // skip files belonging to other git enabled packages
+        if ($EXTRACT(filename, 1, $LENGTH(packageRoot)) = packageRoot) continue
```

Comment:
I think this is wrong. Should get context and packageRoot outside of the loop, and continue if one of the following is true:
* context.Package is an object and filename is not under the package root
* context.Package is not an object and filename is not under the namespace-default root

Note, this might have side effects in the WebUI too - you'll need to make sure that the endpoint that lists uncommitted items properly filters to the current package context too. (Just something to test: if you have uncommitted changes from multiple package contexts, what shows in the web UI in each of those contexts?)

Reply:
Actually, `##class(SourceControl.Git.Utils).TempFolder()` already covers all this:

```
ClassMethod TempFolder() As %String
{
    set context = ##class(SourceControl.Git.PackageManagerContext).%Get()
    if context.IsInGitEnabledPackage {
        quit context.Package.Root
    }
    quit ..DefaultTempFolder()
}
```